frontend/src/pages/Profile.js

```diff
@@ -1,15 +1,303 @@
-import React from 'react';
+import React, {useContext, useEffect, useState} from 'react';
 import Alert from 'react-bootstrap/Alert'
+import Modal from "react-bootstrap/Modal";
+import ErrorMessage from "../messages/errorMessage";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import {AuthContext} from "../context/authContext";
+import Card from "react-bootstrap/Card";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import TogglePage from "../components/TogglePage";
+
 
 export default () => {
+    const {user, setUser} = useContext(AuthContext);
+    const [states, setStates] = useState([]);
+    const currentCustomerId = localStorage.getItem("currentCustomerId") != null ? localStorage.getItem("currentCustomerId") : 0;
+
+    const [addressDto, setAddressDto] = useState({
+        city: '',
+        state: null,
+        addressLine1: '',
+        addressLine2: ''
+    });
+    const [userDto, setUserDto] = useState({
+        id: '',
+        name: '',
+        surname: '',
+        birthday: '',
+        addressDto:'',
+        role: '',
+        password:'',
+        username: '',
+        email: ''
+    });
+    const [state, setState] = useState({
+        id: '',
+        state: ''
+    });
+    const [validError, setError] = useState([]);
+    const [errorMessage, setErrors] = useState('');
+    const handleName = (e) => {
+        setUserDto(preState => ({
+            ...preState,
+            name: e.target.value
+        }));
+    };
+    const handleSurname = (e) => {
+        setUserDto(preState => ({
+            ...preState,
+            surname: e.target.value
+        }));
+    };
+
+    const handlePassword = (e) => {
+        setUserDto(preState => ({
+            ...preState,
+            password: e.target.value
+        }));
+    };
+
+
+    const handleBirthday = (e) => {
+        setUserDto(preState => ({
+            ...preState,
+            birthday: e.target.value
+        }));
+    };
+
+
+    useEffect(() => {
+
+    fetch("http://localhost:8080/customers/" + currentCustomerId + "/users/username/" + user.username)
```

Comment:
We may use es6 string concatenation here.
https://developers.google.com/web/updates/2015/01/ES6-Template-Strings